### ISSUE TITLE
make_hkaman makes a seperate aman for each 372 feed

### ISSUE THREAD

**Issue by skhrg** [simonsobs/sotodlib#446]
Tried loading up some LATR thermometry data with the new code in `hk_utils` that was merged recently and I noticed that every feed from a Lakeshore 372 gets put into a separate sub-axis manager. This does make since because of the way that the 372's sample but from a UX perspective its not ideal.

Would it be possible to either:
- Treat the 372s as a special case and bin/interpolate them so the feeds are cosampled
- Add in some sort of flag/config option to tell `get_hkaman` to bin/interpolate a subset of feeds
- Something other solution?

Don't think this is that big of a deal since it isn't _technically_ doing anything incorrect, but it does make things a bit clunky.

**Comment by sanahabhimani** [comment]
Yeah! Currently, the way I do detector data with HK data cosampled with it, there is a way to generalize this process such that you could interpolate data outside of detector purposes. Standby! Will prob get it done by end of week.
